Resources/Storage/GFAL2_XROOTStorage.py

```diff
@@ -42,10 +43,10 @@ def __init__( self, storageName, parameters ):
     :param str wspath: location of SRM on :host:
     """
     # # init base class
-    super( GFAL2_XROOTStorage, self ).__init__( storageName, parameters )
+    super(GFAL2_XROOTStorage, self).__init__(storageName, parameters)
     self.srmSpecificParse = False
 
-    self.log = gLogger.getSubLogger( "GFAL2_XROOTStorage", True )
+    self.log = LOG.getSubLogger("GFAL2_XROOTStorage")
```

Comment:
```suggestion
    self.log = LOG.getSubLogger(storageName)
```
`LOG` already has GFAL2_XROOTStorage from the Module in its path